src/modules/enroll_activity/app/enroll_activity_usecase.py

```diff
@@ -30,34 +30,31 @@ def __call__(self, user_id: str, code: str) -> Enrollment:
         if user is None:
             raise NoItemsFound('User')
 
-
         enrollment = self.repo.get_enrollment(user_id=user_id, code=code)
 
-        if type(enrollment) is not None:
+        if enrollment is not None:
             if enrollment.state == ENROLLMENT_STATE.ENROLLED:
```

Comment:
eu tirava esse if

Reply:
porque se não for None obrigatoriamente é forbidden, não?

Reply:
Não da pra fazer None.state, por isso tem esse "if"